src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/adsbase/AdsOoyalaObject.java

```diff
@@ -107,4 +136,21 @@ private void waitForColorAds(WebElement element, Color color) {
   private void logMessage(Color color, int duration) {
     PageObjectLogging.log("Video", "Video content had " + color + " during " + duration + " seconds", true);
   }
+
+  public void waitForAdStartsPlaying() {
+    wait.forElementVisible(AD_LAYER_SELECTOR);
+  }
+
+  public void allowToPlayVideoForSomeTime(Duration duration) {
+    try {
+      Thread.sleep(duration.toMillis());
+    } catch (InterruptedException e) {
+      e.printStackTrace();
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Use a logger to log this exception. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AS1148)